setup.cfg

```diff
@@ -55,8 +55,8 @@ console_scripts =
 cibuildwheel = resources/*
 
 [flake8]
-extend-ignore = E203,E501,B950,B023
-extend-select = B9
+extend-ignore = E203,E501,B023,B028
+extend-select = B902,B903,B904,B906
```

Comment:
This is simpler and forward compatible. It will be possible to remove B028 once it gets renamed into B907 (an optional check). I've moved some of the strings to using the !r suggestion, but it was incorrect for a few of them or changed the output of TOML files (not in a bad way, but would be some churn), so we should avoid requiring this (unless it becomes smarter, like checking for strings or checking for double quotes).

Reply:
I've removed the now-renamed B028 and required 23.1.17+ in the pre-commit file.